cvat/apps/dataset_manager/bindings.py

```diff
@@ -513,19 +575,34 @@ def categories(self):
         return self._categories
 
     @staticmethod
-    def _load_categories(cvat_anno):
+    def _load_categories(cvat_anno, dimension):
         categories = {}
 
         label_categories = datumaro.LabelCategories(attributes=['occluded'])
 
+        user_info = {}
+        if dimension == DimensionType.DIM_3D:
+            user_info = {"name": cvat_anno.meta['task']['owner']['username'],
+                         "createdAt": cvat_anno.meta['task']['created'],
+                         "updatedAt": cvat_anno.meta['task']['updated']}
+        index = 0
         for _, label in cvat_anno.meta['task']['labels']:
+            attributes = []
             label_categories.add(label['name'])
+
             for _, attr in label['attributes']:
-                label_categories.attributes.add(attr['name'])
+                if dimension == DimensionType.DIM_3D:
```

Comment:
How it depends on the task dimension?

Reply:
This check is not required now.
Reverted changes